swap/peer.go

```diff
@@ -202,11 +203,15 @@ func (p *Peer) sendCheque() error {
 		return fmt.Errorf("error while saving pending cheque: %v", err)
 	}
 
-	err = p.updateBalance(int64(cheque.Honey))
+	honeyAmountInt64 := int64(cheque.Honey)
+	err = p.updateBalance(honeyAmountInt64)
 	if err != nil {
 		return fmt.Errorf("error while creating cheque: %v", err)
 	}
 
+	metrics.GetOrRegisterCounter("swap.cheques.emitted.num", nil).Inc(1)
+	metrics.GetOrRegisterCounter("swap.cheques.emitted.total", nil).Inc(honeyAmountInt64)
```

Comment:
it's not immediately clear what the difference between these 2 is until you look at the params.

maybe the second should include words like `honey`; `total` could be interpreted as "number of cheques", which is supposed to be the first one.